lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -210,17 +211,27 @@ def test_no_ccx_created(self):
     def test_create_ccx(self):
         """
         Create CCX. Follow redirect to coach dashboard, confirm we see
-        the coach dashboard for the new CCX.
+        the coach dashboard for the new CCX. Also, confirm that the coach
+        has staff access to it.
         """
+
         self.make_coach()
         url = reverse(
             'create_ccx',
-            kwargs={'course_id': self.course.id.to_deprecated_string()})
+            kwargs={'course_id': unicode(self.course.id)})
+
         response = self.client.post(url, {'name': 'New CCX'})
         self.assertEqual(response.status_code, 302)
         url = response.get('location')  # pylint: disable=no-member
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+
+        pattern = re.compile(r"http://[^\/]+/courses/([^\/]+)/ccx_coach")
+        results = re.match(pattern, url)
+        ccx_key = results.group(1)
```

Comment:
@ormsbee could you do a quick drive-by here? I don't think regular expressions can possibly be the best way to get the ccx key out of this URL.